### DISCUSSION TITLE
Use vpn only on device (do not share vpn to connected clients)

### DISCUSSION THREAD

**Discussion by PallavRastogi** [Mygod/VPNHotspot#307] [Q&A]
I want to just use this app as a wifi repeater, How can I make sure that only the mobile's connection is through vpn but all connected devices access internet without vpn?
Also does enabling ipv6 routing increase internet speed and/or stability?

**Comment by Mygod** [comment]
Put `none` in upstream interfaces and leave blank for fallback interfaces. Alternatively bypass this app in your VPN app settings.
